### ISSUE TITLE
New Chrome UI?

### ISSUE THREAD

**Issue by ODB88** [Eloston/ungoogled-chromium#563]
Hi,

I really want to use this version of chrome...right now im using chromium but want to use something that is focused on privacy and Firefox is not working great on Mac OS.

However, I've been following the development of UC for quite some time and trying it with every new release and i cant get the new Chrome UI. 

I try to activate it via //flags but is not working. 

Is there anything i can do or i just need to wait?

Thanks

**Comment by intika** [comment]
i don't know about mac may be @pastmaster007 can let you know... but for linux new design is enabled automatically since v69

**Comment by ODB88** [comment]
I tried 68 and 69 from the latest builds, maybe should i remove the cache or anything like that?

**Comment by intika** [comment]
@ODB88 please close the issue

**Comment by hamishknight** [comment]
And in 70.0.3538.77 you now get the new UI by default.

**Comment by stfnhh** [comment]
Any way to hide the profile icon?

**Comment by hamishknight** [comment]
@stfnhh You can change the flag `#show-avatar-button` for that.

**Comment by intika** [comment]
> Any way to hide the profile icon?

yes this `#show-avatar-button` is a new feature added by @qvint in v70 
just browse chrome://flags
